doc/03-omnifest/01-directive.md

```diff
@@ -73,7 +73,7 @@ otk.include: ${variable}
 ```
 
 If a `${}` appears in a `str` value then its string value as it appears
-in `otk.define` is replaced into the string. Note that using the sugared form
+in `otk.define` is replaced into the string. Note that substitutions
 in this form requires the value to be a string in the `otk.define`.
```

Comment:
```suggestion
in `otk.define` is replaced into the string. Note that substitutions
in this form require the value to be a string in the `otk.define`.
```

Reply:
Hm, hm, I'm not a native speak but to me both sound okay? Fine committing this though (as both sound okay to me :)